tools/model-compiler/src/main/java/io/spine/tools/compiler/annotation/ModuleAnnotator.java

```diff
@@ -95,24 +96,26 @@ private OptionJob(ApiOption protobufOption, ClassName javaAnnotation) {
 
         @Override
         public void execute(AnnotatorFactory factory) {
-            _debug("Annotating sources marked as `{}` with `{}`.", protobufOption, javaAnnotation);
-            _debug("Annotating by the file option.");
+            FluentLogger.Api debug = _debug();
+            debug.log("Annotating sources marked as `%s` with `%s`.", protobufOption,
+                      javaAnnotation);
```

Comment:
Don't we have enough space for this parameter? If so, I'd better move both parameters to the second line